src/libclipper/include/clipper/app_metrics.hpp

```diff
@@ -0,0 +1,47 @@
+#ifndef CLIPPER_APP_METRICS_HPP
```

Comment:
I think we should include this within `metrics.hpp` and the `metrics` namespace instead.